contracts/gateway/AxelarAmplifierGateway.sol

```diff
@@ -53,6 +53,7 @@ contract AxelarAmplifierGateway is BaseAmplifierGateway, BaseWeightedMultisig, U
     function _setup(bytes calldata data) internal override {
         (address operator_, WeightedSigners[] memory signers) = abi.decode(data, (address, WeightedSigners[]));
 
+        // operator is an optional parameter, it can also be removed in the future to make ownership completely decentralized
```

Comment:
the comment about the owner being able to remove / transfer operator to address(0), should be made on the `transferOperatorship` method

Reply:
add a unit test for this if there isn't one